pkg/k8s/labels.go

```diff
@@ -173,6 +173,10 @@ const (
 	// volume mounted into each proxy to store identity credentials.
 	IdentityEndEntityVolumeName = "linkerd-identity-end-entity"
 
+	// IdentityIssuerSecretName is the name assigned the temporary end-entity
+	// volume mounted into each proxy to store identity credentials.
+	IdentityIssuerSecretName = "linkerd-identity-issuer"
```

Comment:
templatize this string in `identity.yaml`

Reply:
We're not really consistent about this, and I don't feel it buys us anything.